packages/web/docs/src/pages/docs/gateway/deployment/resources.mdx

```diff
@@ -0,0 +1,35 @@
+import { Callout } from '@theguild/components'
+
+# Resources
+
+Finding and defining the right amount of resources for your GraphQL gateway is important to ensure
+ongoing performance and reliability. This guide will help you understand how to define the resources
+for your gateway.
+
+## CPU and Memory
+
+The two main resources you need to define for your gateway are CPU and memory. These resources are
+used to ensure that your gateway has enough resources to run efficiently.
+
+For most applications, you can with the following default values:
+
+- **CPU: 1**
+- **Memory: 1GB**
+
+These values are a good starting point for most applications. However, you may need to adjust these
+values based on your application's requirements. For example, if your application has a high volume
+of traffic, you may need to increase the CPU and memory resources.
+
+For production environments with high-traffic, we recommend using the following values:
+
+- **CPU: 2**
+- **Memory: 2-4GB**
+
+## Environment Variables
+
+We recommend setting `NODE_ENV` to `production` to ensure that your gateway runs in production mode.
+This will also enable parallelism optimizations that can improve performance.
+
+By default, the gateway will available parallelism for the amount of workers to spawn. You can set
```

Comment:
The wording is a bit awkward here.

Reply:
Ah, yeah, wrong copy/paste. Pushed a fix, and also fixed typos. Thanks!